### DISCUSSION TITLE
IPTV per user

### DISCUSSION THREAD

**Discussion by UltimateMedia** [jellyfin/jellyfin#7019] [General]
I was hoping there would be a future update where i can assign a unique m3u iptv playlist per user. I have like half my family using my jellyfin server but i would like for all the accounts to have their own separate and unique m3u list. So my provider won't ban my acocunts for over usage. And so i can even set up which channels can be seen and used per user. 

Like i said my brother last night, when Jellyfin gets this im done, i have my ultimate media solution. Finaly

**Comment by Waradmiraal** [comment]
This would be a great idea!

**Comment by gerbenp** [comment]
Would be great if this idea will be added, since there are servers which does only 1 connection per IP and other users using Jellyfin.

**Comment by freshman26** [comment]
Any update on this or someone that managed to figure it out? Would be great to assign a m3u per user(s) to not risk being banned or have different iptv's based on preferences.
